src/index.css

```diff
@@ -478,3 +484,13 @@ ACCESSIBILITY - Also see any other focus styles
   outline: 2px solid rgba(0, 121, 116, 1);
   outline-offset: 1px;
 }
+
+/* Semantic UI Default Override */
+
+  .mini.ui.steps a.step:hover {
+  background: #dcddde;
+}
+
+    div[role="option"].active.selected.item:hover {
```

Comment:
This only changes the hover color for the active and selected item. The non-active/selected item(s) don't get the same hover color. Is there a specific reasoning behind that?

Reply:
The effect seen here is my interpretation of the description as described in the issue, namely that the hover effect only needs to be added to the currently selected item on hover, with no other changes.

Reply:
If you're talking about what @2001zhaozhao described, I can understand the confusion. If you want, we can merge this based on that.

I think his intention, though, was the 'hover' state. He also only described form nav bar menu styles that way, but I think that was meant to be on 'hover' as well, which is the way you implemented it.

The confusion in terms might be because he's not super familiar with semantic-ui classes or CSS. @2001zhaozhao : Whatever your intention was, no worries, you input has been really useful in moving this forward 👍

Reply:
Ah, you've changed that form nav bar interaction now. Well, if you want, we can accept both of them and we can discuss the idea of applying it to all hovered elements.

Reply:
Hmm, no, something different happened. I think the styles aren't being applied anymore at all.

Reply:
We probably should discuss this in person Tuesday night, I'm definitely seeing the changes I applied still working.